daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -255,14 +257,16 @@ final class Engine {
       submitters: Set[Party],
       commands: ImmArray[SpeedyCommand],
       ledgerTime: Time.Timestamp,
-      transactionSeedAndSubmissionTime: Option[(crypto.Hash, Time.Timestamp)]
-  ): Result[(Transaction.Transaction, Boolean)] = {
+      submissionTime: Time.Timestamp,
```

Comment:
Can you update the comment? We no longer have [[transactionSeed]] here anymore.

Reply:
fixed